### DISCUSSION TITLE
Nano ELS H4 Availability

### DISCUSSION THREAD

**Discussion by Festivejelly** [kachurovskiy/nanoels#177] [General]
Dismayed to see the H4 not available on the store anymore. Ive just got a new lathe and was hoping to add a H4 to this one too.

If you're not selling them anymore would DIY making one be difficult, im not hugely familiar with electronics but ive made some basic circuits in the past.

Your ELS system is the best if used and the best ive seen available anywhere. Especially with the lathecode and ability to send gcode.

**Comment by vegetate7** [comment]
You can order pre-soldered boards from JLCPCB/LCSC. Or partially pre-soldered, let's say only small SMDs.

**Reply by Festivejelly** [reply]
Oh okay so I guess id need to upload the various files. I really dont fancy doing the soldering so I wonder if I could get a pick and place option to full make and assemble the board.

**Comment by kachurovskiy** [comment]
Yeah, sorry about that. Perhaps this will provide an opportunity for someone in the community to take on the hardware distribution.

Ordering the required parts is not trivial since Gerber files reference components that have since gone out of stock too. JLCPCB allows swapping out one part type for the other when ordering so that helps a bit. There are however some changes that need to be made to Gerber mentioned in https://github.com/kachurovskiy/nanoels/blob/main/h4/parts.md to avoid extra manual work afterwards.

Some people are working on alternative hardware implementations, I will be highlighting any such projects when they are ready.
